ViewModel/Hydraulics/GriddedData/GriddedImporterVM.cs

```diff
@@ -166,36 +154,18 @@ public void FileSelected(string fullpath)
                         importResult.AddErrorMessage(result.ErrorMessage);
                     }
                 }
-
-                string errorMsg = " The selected directory must have 8 subdirectories that each contain one .vrt file and at least one .tif file.\n";
-
-                //we require 8 valid directories
-                if (validDirectories.Count < 8)
+                string errorMsg = "The selected directory must have at least 1 subdirectories that each contains one .tif file.\n";
+                double prob = 0;
+                foreach (string dir in validDirectories)
                 {
-                    string dirName = Path.GetFileName(fullpath);
-                    importResult.InsertMessage(0, "Directory '" + dirName + "' did not contain 8 valid subdirectories." + errorMsg);
-                    MessageBox.Show(importResult.ErrorMessage, "Invalid Directory Structure", MessageBoxButton.OK, MessageBoxImage.Error);
+                    prob += .1;
+                    AddRow(Path.GetFileName(dir), Path.GetFullPath(dir), prob);
                 }
-                else if (validDirectories.Count > 8)
+                //we might have some message for the user?
+                if (!importResult.IsValid)
                 {
-                    string dirName = Path.GetFileName(fullpath);
-                    importResult.InsertMessage(0, "Directory '" + dirName + "' contains more than 8 valid subdirectories." + errorMsg);
-                    MessageBox.Show(importResult.ErrorMessage, "Invalid Directory Structure", MessageBoxButton.OK, MessageBoxImage.Error);
-                }
-                else
-                {
-                    double prob = 0;
-                    foreach (string dir in validDirectories)
-                    {
-                        prob += .1;
-                        AddRow(Path.GetFileName(dir), Path.GetFullPath(dir), prob);
-                    }
-                    //we might have some message for the user?
-                    if (!importResult.IsValid)
-                    {
-                        importResult.InsertMessage(0, "The selected directory contains 8 valid subdirectories and will ignore the following:\n");
-                        MessageBox.Show(importResult.ErrorMessage, "Valid Selection", MessageBoxButton.OK, MessageBoxImage.Information);
-                    }
+                    importResult.InsertMessage(0, "The selected directory contains 8 valid subdirectories and will ignore the following:\n");
```

Comment:
Should this message still say 8? 
Maybe change to: The selected directory contains valid subdirectories and will ignore the following